eve/images/pep8-checker/Dockerfile

```diff
@@ -5,7 +5,15 @@
 
 FROM alpine:3.8
 
-RUN apk add --no-cache git python python3 py2-pip py-twisted && \
+RUN apk add --no-cache \
+        gcc \
+        git \
+        musl-dev \
+        python \
+        python2-dev \
+        python3 \
+        py2-pip \
+        && \
     pip install tox && \
```

Comment:
Could we fix the version here ? == 1.8.0 seems a good version here (sorry my bad, to not have the correct version pinned here), then checking if py-twisted work or if gcc and musl is needed (and in this case remove py-twisted)

Reply:
Done